application/transaction/transaction_test.go

```diff
@@ -5,32 +5,30 @@ import (
 
 	"github.com/fabric8-services/fabric8-cluster/application/transaction"
 	"github.com/fabric8-services/fabric8-cluster/gormtestsupport"
-	"github.com/fabric8-services/fabric8-cluster/resource"
 
 	"github.com/pkg/errors"
 	"github.com/stretchr/testify/require"
 	"github.com/stretchr/testify/suite"
 )
 
-type TestTransaction struct {
+type TestTransactionSuite struct {
```

Comment:
I would rather call it `TransactionTestSuite` since it is based on the `DBTestSuite`, but it's just a matter of naming...

Reply:
Done in https://github.com/fabric8-services/fabric8-cluster/pull/24/commits/9127aea20f12b3acd503ae8b47efbec3c1852d8d